src/abi.h

```diff
@@ -155,6 +163,7 @@ typedef struct netplan_backend_settings {
     char *stable_id;
     char *device;
     GData* passthrough; /* See g_datalist* functions */
+    guint private_key_flags;
```

Comment:
This should probably move into `NetplanAuthenticationSettings`

Reply:
`NetplanAuthenticationSettings`only has Wifi related configuration. I don't think it should be moved there, it's more like we'll need different `private_key_flags` variables in different places for each key we want to correlated the flags with.

This one in particular should probably be moved to `netplan_net_definition.tunnel`.

Reply:
Yes, you're right!
Also, we need to take special care about the `netplan_net_definition.tunnel` struct (and `struct NetplanAuthenticationSettings`, too) as they are part of the bigger `struct netplan_net_definition`. So adding new fields in-between would break ABI.

We probably need to use some quirk as done with `tunnel_ttl`, adding a new field to the end of the top-level struct (+ a relevant XXX/TODO comment), and refactor it eventually, once we bump the library's SONAME.